demo/admin/vite.config.mts

```diff
@@ -76,7 +76,7 @@ export default defineConfig(({ mode }) => {
             adminPackagesHotReloadPlugin,
         ],
         server: {
-            host: true,
+            host: false,
```

Comment:
I suggest to use the following:
```suggestion
            host: 'localhost',
```
- more visible what it does
- it's the default value (https://vite.dev/config/server-options#server-host)

Reply:
I would leave it, to make it visible that it is only listening on localhost (and you have no potential security problem)

Reply:
c0971663b91e61cc6cdf4579ce6ef8e355929713